lib/client.js

```diff
@@ -92,6 +92,26 @@ api.sendEvent = (event, peer, callback) => {
   });
 };
 
+api.sendVote = (vote, peer, callback) => {
+  const url = peer + '/votes';
+  request.post({url, strictSSL, json: vote}, (err, res) => {
+    if(err) {
+      return callback(new BedrockError(
+        'Could not send vote.', 'NetworkError', {vote, peer}, err));
+    }
+    if(res.statusCode !== 201) {
```

Comment:
We should handle some duplicate error code and ignore it here. It's possible that someone else sends the vote before we do.